recipe/install-libgcc.sh

```diff
@@ -35,9 +35,10 @@ popd
 
 mkdir -p ${PREFIX}/lib
 
-# no static libs
-find ${PREFIX}/${CHOST}/lib -name "*\.a" -exec rm -rf {} \;
-# no libtool files
+# we can't remove static base libraries, as it would make option -static not working
+# and things like qt won't build anymore.
```

Comment:
Aren't these static libraries in `libgcc-devel`? (At least that was the intention)